public/stylesheets/site/2.0/18-zone-searchbrowse.css

```diff
@@ -78,6 +78,10 @@ form.filters dl {
   padding: 0;
 }
 
+.filters dt, .filters select {
```

Comment:
Our select menus and form dts usually have a minimum width of 10.5em set, so they were getting cut off when the filters got narrower:

![Very narrow filters](https://cloud.githubusercontent.com/assets/907055/7668088/9a35f1f4-fbf3-11e4-8e95-75ebf3ea4ecd.png)

Now the text will wrap onto the next line, so you can still see everything when the filters are narrow.